wgsl/extract-grammar.py

```diff
@@ -510,7 +761,321 @@ def read_spec(options):
                     scanner_i += scanner_parse[-1]  # Advance line index
         scanner_i += 1
 
-    result[scanner_rule.name()]["_comment"] = [["`/\\/\\/.*/`"]]
+    RULE_NAME_CODE_POINTS = string.ascii_lowercase + string.digits + "_"
```

Comment:
This doesn't belong in read_spec anymore, because it's both reading data from another place *and* writing files.
Make different functions for those two pieces, and invoke them separately from read_spec.